include/OpenColorIO/OpenColorIO.h

```diff
@@ -2834,51 +2838,7 @@ class OCIOEXPORT GpuShaderCreator
                               Interpolation interpolation,
                               const float * values) = 0;
 
-    // TODO: Move to .rst
-    // !rst:: Methods to specialize parts of a OCIO shader program.
-    //
-    // **An OCIO shader program could contain:**
-    //
-    // 1. A declaration part  e.g., uniform sampled3D tex3;
-    //
-    // 2. Some helper methods
-    //
-    // 3. The OCIO shader function may be broken down as:
-    //
-    //    1. The function header  e.g., void OCIODisplay(in vec4 inColor) {
-    //    2. The function body    e.g.,   vec4 outColor.rgb = texture3D(tex3, inColor.rgb).rgb;
-    //    3. The function footer  e.g.,   return outColor; }
```

Comment:
@michdolan, I think the Python stuff has trouble with numbered lists -- the build failed until I changed these to unnumbered lists. I'm sorry, I don't have the traceback available.

Reply:
Ok, I'll need to have look at the doxygen extraction code. Should be fairly easy to add.